### ISSUE TITLE
Train on CULane

### ISSUE THREAD

**Issue by khw321** [MaybeShewill-CV/lanenet-lane-detection#20]
@MaybeShewill-CV  Hello, I want to train your model on CULane dataset, which has 88880 for training set, 9675 for vali-dation set, and 34680 for test set, and some images without lines. Could you give some advice on training parameters?

**Comment by MaybeShewill-CV** [comment]
@khw321 You could try the parameters used in tusimple dataset first.

**Comment by khw321** [comment]
@MaybeShewill-CV I am concerned about the epochs size 94000 with batch size 4 may be small for the CULane dataset, and I have found 'culane' in your training code, whether if you had trained on CULane dataset?
btw, how to train on multi-GPU?
Thank you :)

**Comment by MaybeShewill-CV** [comment]
@khw321 I have not tested the model on multi-GPU the batch size nums is limited to GPU memory

**Comment by khw321** [comment]
@MaybeShewill-CV OK, thanks!
